spec/lib/frigg/persister_spec.rb

```diff
@@ -0,0 +1,483 @@
+# frozen_string_literal: true
+
+return if Hyrax.config.disable_wings
+
+require 'spec_helper'
+require 'valkyrie/specs/shared_specs'
+require 'wings'
+require 'frigg/metadata_adapter'
+
+RSpec.describe Frigg::Persister, :active_fedora do
+  subject(:persister) { described_class.new(adapter: adapter) }
+  let(:adapter) { Frigg::MetadataAdapter.new }
+  let(:query_service) { adapter.query_service }
+
+  xcontext "When passing a Valkyrie::Resource that was never an ActiveFedora::Base" do
```

Comment:
This skipped spec context would go with the commented code in the frigg persister.

Reply:
that's correct, and for the same reason